packages/fast-components/src/anchor/anchor.styles.ts

```diff
@@ -0,0 +1,18 @@
+import { css } from "@microsoft/fast-element";
+import {
+    AccentButtonStyles,
+    BaseButtonStyles,
+    HypertextStyles,
+    LightweightButtonStyles,
+    OutlineButtonStyles,
+    StealthButtonStyles,
+} from "../styles";
+
+export const AnchorStyles = css`
+    ${BaseButtonStyles}
+    ${AccentButtonStyles}
+    ${HypertextStyles}
+    ${LightweightButtonStyles}
+    ${OutlineButtonStyles}
+    ${StealthButtonStyles}
```

Comment:
I thought we were trying to avoid CSS bloat per the spec, did we decide this was no longer necessary?

Reply:
@EisenbergEffect proposed a registration methodology which would support minimizing bloat incurred by this by being able to register and unregister styles when used so we can avoid having 5 different components. The thread is here: https://github.com/microsoft/fast-dna/pull/2844#discussion_r398941666